### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by asauber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +36,95 @@ Want to improve the linode-cloud-controller-manager? Please start [here](.github
...
+$ IMG=linode/linode-cloud-controller-manager:canary make docker-build
 
+# Push Image
+$ IMG=linode/linode-cloud-controller-manager:canary make docker-push
+```
 
+Then, to run the image
+```bash
+$ docker run -ti linode/linode-cloud-controller-manager:canary
+```
```
[the remark this answers is not in the document]
Comment (asauber) [reply README.md]: Those sections exist already.

File: README.md [hunk 2]
```diff
@@ -24,9 +36,95 @@ Want to improve the linode-cloud-controller-manager? Please start [here](.github
...
+[Kubernetes Cloud Controller Manager](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/).
+
+## Upstream Developer Documentation
+
+[Developing a Cloud Controller Manager](https://kubernetes.io/docs/tasks/administer-cluster/developing-cloud-controller-manager/).
+
+## Development Guide
+
+This document is intended to be the canonical source of truth for things like
+supported toolchain versions for building the Linode Cloud Controller
```
[the remark this answers is not in the document]
Comment (asauber) [reply README.md]: right, removed them

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sarahelizgray) [reply README.md]: Would you mind having them appear at the bottom of the readme, in keeping with our other repos?

[the remark this answers is not in the document]
Comment (asauber) [reply README.md]: Ok, that makes sense. Keep the main content repo-specific.

**Comment by sarahelizgray** [comment]
@asauber changes approved. Nice work here :)

**Comment by asauber** [comment]
Thanks @sarahelizgray, merging

**Pull request merged**